### PR TITLE
Update cortana-integration.md

### PR DESCRIPTION
**Pull request by msbemba**
Added details about requirements for briefing email and Teams.

### CODE REVIEW

**Comment by msbemba** [comment]
@JohanFreelancer9 Please copy review

**Comment by msbemba** [comment]
issue#https://github.com/MicrosoftDocs/microsoft-365-docs/issues/4920

File: microsoft-365/admin/misc/cortana-integration.md [hunk 1]
```diff
@@ -77,6 +77,9 @@ Beginning with Windows 10, version 2004, Cortana is a Universal Windows Platform
 
 ### Cortana voice assistance in Teams
 
+> [!Note]
```
Comment (JohanFreelancer9) [inline microsoft-365/admin/misc/cortana-integration.md]:
```suggestion
> [!NOTE]
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review - minor edit.

**Comment by yogkumgit** [comment]
@msbemba

**Pull request merged**